Core/Src/spi_command_handler.c

```diff
@@ -216,6 +213,28 @@ int step_transfer() {
     return 1;
 }
 
+void take_picture() {
```

Comment:
This should be refactored to take VIS or NIR as arguments, at least for the set up. I guess the point is that VIS and NIR images are captured exactly at the same time (just hoping it's not to save opcodes or functions)